staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo.go

```diff
@@ -259,6 +259,9 @@ func WithRequestInfo(parent context.Context, info *RequestInfo) context.Context
 
 // RequestInfoFrom returns the value of the RequestInfo key on the ctx
 func RequestInfoFrom(ctx context.Context) (*RequestInfo, bool) {
+	if ctx == nil {
+		return nil, false
```

Comment:
I'm wondering where this nil ctx is coming from.
I think we should try to understand that at least and ideally fix the root cause (otherwise, I'm afraid we may hit similar issues in the future, because there are number of places relying on non-nil context).

Reply:
> I'm wondering where this nil ctx is coming from.

sadly we could not reproduce this, it happened (literally accidently i guess) on one of our stable master servers